keyboards/handwired/ms_sculpt_mobile/teensypp/teensypp.h

```diff
@@ -0,0 +1,2 @@
+#pragma once
```

Comment:
This file needs a GPLv2-compatible license header.

* [PR Checklist - Example GPLv2 Header - QMK Firmware Documentation](https://docs.qmk.fm/#/pr_checklist?id=example-gplv2-header)